slsim/lens.py

```diff
@@ -296,7 +297,7 @@ def einstein_radius_deflector(self):
                     float(self.deflector.velocity_dispersion(cosmo=self.cosmo))
                 )
             else:
-                # numerical solution for the Einstein radius
+                # numerical solution for the Einstein radius #TODO check
```

Comment:
is there a way you want to check that or do you want us to check it on our side?

Reply:
As I asked in Slack before, it looks like I can calculate the Einstein radius properly for massive halos (+ galaxies), but for small masses, it returns nan. I'm not sure if this is a bad thing in other calculation parts.

Reply:
@kta-cosmo for small halos, there may not be a Einstein radius as the lensing convergence never exceeds unity even in the center. We could replace nan with zero in the code for these cases if needed

Reply:
Thank you. Yes, I think so. I'll leave it to you to decide if nan should be changed to zero. I removed the TODO comment.